pkg/manifests/config.go

```diff
@@ -209,7 +209,7 @@ func defaultClusterMonitoringConfiguration() ClusterMonitoringConfiguration {
 				},
 				NetClass: NodeExporterCollectorNetClassConfig{
 					Enabled:    true,
-					UseNetlink: true,
+					UseNetlink: false,
```

Comment:
hmm I missed that we enabled the netlink driver by default in #1894. To avoid any regression/change, I agree that it's better to keep the default as it was before (e.g. using sysfs).